api/ShadowRoot.json

```diff
@@ -83,16 +83,16 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ShadowRoot/delegatesFocus",
           "support": {
             "chrome": {
-              "version_added": "45"
+              "version_added": "53"
```

Comment:
This whole interface [landed in Chrome 57](https://storage.googleapis.com/chromium-find-releases-static/f33.html#f336ae65b6486a7162946053b073ff454c246f8f). (These earlier version numbers are from an experimental implementation Shadow DOM called 'v0'. )